iphone/Maps/Classes/CustomViews/MapViewControls/Search/Console/MWMConsole.mm

```diff
@@ -15,6 +15,9 @@ + (BOOL)performCommand:(NSString *)cmd
 
   if ([self performSound:cmd])
     return YES;
+  
+  if ([self perform3dMode:cmd])
```

Comment:
I guess we have hidden commands only in qt, iOS and Android projects, not in core.